lib/tweet.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './user'
+require 'pg'
+class Tweet
+  attr_reader :message, :username, :id, :time
+
+  def initialize(id:, username:, message:, time:)
+    @id = id
+    @username = username
+    @message = message
+    @time = Time.parse(time).strftime("%Y/%m/%d %k:%M:%S")
+    end
+
+  def self.display_tweets
```

Comment:
You could call it only `self.display`/`self.add` so it's not repetitive tweet, as you are in the class Tweet it will make sense what are you calling to.